gen3utils/manifest/manifest_validator.py

```diff
@@ -145,6 +145,18 @@ def get_manifest_version(manifest_versions, service):
                 return service_version
 
 
+def is_release_tag(parsed_version):
```

Comment:
could you move it to `etl_validator.py ` since it's only used in that file

Reply:
🤦  Sorry, I came back to this today and merged before I saw your comment. Are you ok with the fn staying in this file? I can also make a quick patch moving it.

Reply:
it's fine :)